versions.props

```diff
@@ -22,7 +22,13 @@ org.jmock:jmock = 2.12.0
 org.mockito:* = 5.2.0
 org.slf4j:* = 1.7.36
 io.leangen.geantyref:geantyref = 1.3.4
-io.undertow:* = 2.2.20.Final
+io.undertow:* = 2.2.24.Final
+# dependency-upgrader:OFF
+# Last version of undertow with both Jakarta+Javax support before 'undertow-servlet' was
+# updated to jakarta only. This should NEVER be done in production code, it is only used
+# here to enable existing test coverage.
+io.undertow:undertow-servlet-jakarta = 2.2.20.Final
+# dependency-upgrader:ON
```

Comment:
@carterkozak should we also bump [`com.palantir.conjure.java` to 7.5.0](https://github.com/palantir/conjure-java/releases/tag/7.5.0) in this release? I'm also not sure why we have separate pin for `com.palantir.conjure.java:conjure-lib = 7.4.0`

```
com.palantir.conjure.java:* = 7.5.0
```

Reply:
Good call on the unnecessary `conjure-lib`, I'm not sure where that came from. I don't think conjure-java 7.5.0 is published yet

Reply:
yeah, I got 404s locally for `com.palantir.conjure.java:conjure-lib:7.5.0` just a second ago. Will +1 and defer to you if you want to wait for the publish to complete before merging this & cutting CJR release as well

Reply:
Removing the duplication causes consistent CI failures. I'm going to undo that piece and investigate later on.